pinc/CharSuites.inc

```diff
@@ -13,7 +13,7 @@ class PickerSet
     /** @var string[] */
     private array $titles;
 
-    /** @param (?string[])[] $codepoints */
+    /** @param array<int, list<string|null>> $codepoints */
```

Comment:
Interesting, the parse for the previous syntax changed. This used to mean `list<list<?string>>` but it is now `list<list<string>|null>`.

We should be careful with using `array<int, ...>` as it could represent either a map or a vector/list. While the difference is minimal, it is important. Thus this suggestion:

```suggestion
    /** @param list<list<?string>> $codepoints */
```

Feel free to use `string | null` if you don't like the shorthand `?string` (though fair warning, we use it across the codebase).

Reply:
Good suggestion - made this and the other set of changes you suggested. Thanks for the review!